apps/client/src/pages/home/Home.page.tsx

```diff
@@ -36,30 +40,65 @@ export function Home(): ReactElement {
         onTokenReceived: () => navigate(`/${roomId}/room`),
       });
     } else {
-      // TODO: Create tournament in redux store
-      const tournamentId = getTournamentId();
+      // TODO: should receive tournament id(room id) from server
+      const roomId = getRoomId();
       initializeSocket({
         playerName,
         roomId,
-        onTokenReceived: () => navigate(`/${tournamentId}/room`),
+        onTokenReceived: () => navigate(`/${roomId}/room`),
       });
     }
   };
 
   const [playerName, setPlayerName] = useState<string>('');
+  const [isValidPlayerName, toggleIsValidPlayerName] = useState<boolean>(false);
+  // value of join room button. this value will be a room id
+  const [joinRoomButtonValue, setJoinRoomButtonValue] = useState<string>('');
+  const [isValidJoinRoomButtonValue, toggleIsValidRoomButtonValue] =
```

Comment:
should be named as `isPlayerNameValid` and `isJoinRoomButtonValueValid;

Reply:
326958a300c64dc5f91b45f4ac5a6f62e2f2ecda